src/views/PlanTargets.vue

```diff
@@ -0,0 +1,201 @@
+            </b-field>
+          </p>
+        </div>
+        <div class="field has-addons">
+          <p class="control">
+            <b-field label="Date/Time">
+              <b-datetimepicker 
+                id="dateobs" 
+                v-model="user.dateobs"
+                :timepicker="{ incrementMinutes:30 }"
+                :datetime-parser="(d) => {new Date(d)}"
+                required 
+                inline />
+            </b-field>
+          </p>
+        </div>
+      <b-field grouped>
+        <div v-if="user.localtime !== 'X'">
+          <b-radio name="tzinfo" id="tzinfo" native-value="my" v-model="user.tzinfo" required>My timezone</b-radio>
+          <b-radio name="tzinfo" id="tzinfo" native-value="utc" v-model="user.tzinfo" required> UTC</b-radio>
+          <b-radio name="tzinfo" id="tzinfo" native-value="lcl" v-model="user.tzinfo">Local timezone</b-radio>
+        </div>
+        <div v-else>
+          <b-radio name="tzinfo" id="tzinfo" native-value="my" v-model="user.tzinfo" required>My timezone</b-radio>
+          <b-radio name="tzinfo" id="tzinfo" native-value="utc" v-model="user.tzinfo" required> UTC</b-radio>
+        </div>
+      </b-field>
+        <div class="the-button">
+          <b-field class="buttons">
+            <b-button @click="submitForm" type="submit">Find Easy Targets</b-button>
+          </b-field>
+        </div>
+    </form>
+  </div>
+  <div class="results" id="results">
+    <div class="target-columns">
+      <div v-for="target in targlist" :target="target" :key="target.name">
+        <TargetCard :target="target" />
+      </div>
+    </div>
+  </div>
+</section>
+</template>
+
+<script>
+import Vue from 'vue';
+import axios from 'axios';
+import moment from 'moment';
+import TargetCard from '@/components/TargetCard.vue';
+import list from '../../public/data/easytargets.json';
+import helpers from '@/utils/helpers';
+
+export default {
+  name: 'PlanTargets',
+  components: {
+    TargetCard,
+  },
+  data() {
+    return {
+      site_info: {},
+      easylist: list,
+      target: {},
+      targlist: '',
+      user: {
+        lat1: '',
+        lon1: '',
+        dateobs: new Date(Math.round(new Date().getTime() / 1800000) * 1800000),
+        tzinfo: 'my',
+        offset: '',
+        localtime: '',
+        localoffset: '',
+      },
+    };
+  },
+  created: function() {
+    const url = "https://api.photonranch.org/api/all/config"
+    axios.get(url).then(response => {
+      for (let site in response.data) {
+        Vue.set(this.site_info, site, {
+          latitude: response.data[site].latitude,
+          longitude: response.data[site].longitude,
+          name: response.data[site].name,
+          siteoffset: response.data[site].TZ_database_name
+        })
+      }
+      Vue.set(this.user, 'localtime', this.site_info.mrc.siteoffset)
+      Vue.set(this.user, 'lat1', this.site_info.mrc.latitude)
+      Vue.set(this.user, 'lon1', this.site_info.mrc.longitude)
```

Comment:
In reference to my previous comment, I guess this could be just 
`this.localtime = this.site_info.mrc.siteoffset`, etc 
if not nested under the 'user' object.